app/templates/index.hbs

```diff
@@ -7,6 +7,29 @@
   <a href="https://guides.emberjs.com/release/tutorial/ember-cli/" class="margin-left-small">Read the Tutorial</a>
 </EsPageHeader>
 
+<section aria-labelledby="section-ecosystem" class="bg-muted">
+  <div class="container text-center">
+    <div class="row">
```

Comment:
Just wondering, where did you find the docs for using "row" here? I thought we had actually removed the documentation for that (and the implementation) 🤔 

I would have expected you to use the classes described here https://deploy-preview-145--ember-styleguide.netlify.com/concepts/layout

Reply:
Just for the record, part of getting you involved in the process for such a "simple" task is to verify the holes in the documentation. I'm not trying to nit-pick when asking "where did you find this info", that's actually mostly the point of this particular task 😄

Reply:
My bad, I started working on it and since none of the other sections seemed to have the exact layout "my" section needed, I realized we use Bootstrap (or I thought we did) and used that.

I forgot that you also mentioned the Ember guides in our chat.

I'll amend the PR to use the appropriate layout from the styleguide.

Reply:
The first section in the layouts guides was exactly what I needed 🤦‍♂ , I just added a `container` so that it looks more like the design.

Anything else I should fix?

Reply:
Forgot to include screenshots so that you don't have to check out the branch to see what it looks like.

[This one](https://cl.ly/b799d2173e39) is with `class="container"` and [this one](https://cl.ly/fef431db78ee) is without. I think the one that uses container looks more like the design (thought not exactly like it).

Reply:
It's not such a problem to include screenshots because we have the netlify preview 😄 

I'm just going to make one tweak to it and merge your PR now 👍